res/css/rethemendex.sh

```diff
@@ -8,7 +8,7 @@ cd `dirname $0`
     # we used to have exclude /themes from the find at this point.
     # as themes are no longer a spurious subdirectory of css/, we don't
     # need it any more.
-    find . -iname _\*.scss | fgrep -v _components.scss | LC_ALL=C sort |
+    find . -iname _\*.scss | sort | fgrep -v _components.scss | LC_ALL=C sort |
```

Comment:
Looks like there's a change to the `_components.scss` which needs to be committed here:
![image](https://user-images.githubusercontent.com/1190097/78047113-381b5e00-7335-11ea-91eb-6f1640e13673.png)

Reply:
I'd assumed the autogen was done by yarn. Comited, and removed the useless sort.